src/main/resources/assets/minestuck/blockstates/glowing_purple_brick_stairs.json

```diff
@@ -0,0 +1,209 @@
+{
+  "variants": {
```

Comment:
I think it's preferrable to have a more compact format in situations like these (compare to other blockstate files)
Not important enough to warrant a change unless you'd like to fix it, but keep it in mind for the future